### PR TITLE
ticdc: add description about gc-ttl

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Added more documents to describe the usage of gc-ttl from the point of a user, with a more intuitive way.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/7041
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by overvenus — APPROVED** [review]
LGTM, thanks!

**Review by asddongmen — APPROVED** [review]

File: ticdc/troubleshoot-ticdc.md [hunk 1]
```diff
@@ -139,13 +139,27 @@ cdc cli changefeed update -c <changefeed-id> --sort-engine="unified" --sort-dir=
...
+
+- The maximum time the GC safepoint will be retained at the PD when the TiCDC service is stopped.
+- The maximum time a replication task can be suspended when interrupted or manually stopped. If a replication task is suspended longer than the value set by `gc-ttl`, the replication task will enter the `failed` state, cannot be resumed, and will not continue to affect the progress of the GC safepoint.
+
+The second behavior above is introduced in TiCDC v4.0.13 and later versions. The purpose is to prevent a replication task in TiCDC from suspending for too long, causing the GC safepoint of the upstream TiKV cluster not to continue for a long time and retaining too many outdated data versions, thus affecting the performance of the upstream cluster.
+
+> **Note**
+> In some scenarios, such as when using TiCDC for incremental replication after full replication with Dumpling/BR, the default 24 hours of `gc-ttl` may not be sufficient. You need to specify an appropriate value for `gc-ttl` when you start the TiCDC server.
```
Comment (Liuxiaozhen12) [inline ticdc/troubleshoot-ticdc.md:154]:
```suggestion
> **Note:**
>
> In some scenarios, for example, when you use TiCDC for incremental replication after full replication with Dumpling/BR, the default 24 hours of `gc-ttl` may not be sufficient. You need to specify an appropriate value for `gc-ttl` when you start the TiCDC server.
```
这个对应的中文也少一行，还请@hfxsd 再提个 PR 改一下中文吧~

File: ticdc/troubleshoot-ticdc.md [hunk 2]
```diff
@@ -139,13 +139,27 @@ cdc cli changefeed update -c <changefeed-id> --sort-engine="unified" --sort-dir=
...
+If a replication task starts after the TiCDC service starts, the TiCDC owner updates the PD service GC safepoint with the smallest value of `checkpoint-ts` among all replication tasks. The service GC safepoint ensures that TiCDC does not delete data generated at that time and after that time. If the replication task is interrupted, or manually stopped, the `checkpoint-ts` of this task does not change. Meanwhile, PD's corresponding service GC safepoint is not updated either.
+
+If the replication task is suspended longer than the time specified by `gc-ttl`, the replication task enters the `failed` state and cannot be resumed. The PD corresponding service GC safepoint will continue.
```
Comment (Liuxiaozhen12) [inline ticdc/troubleshoot-ticdc.md:160]:
```suggestion
If the replication task is suspended longer than the time specified by `gc-ttl`, the replication task enters the `failed` status and cannot be resumed. The PD corresponding service GC safepoint will continue.
```

**Review by Liuxiaozhen12 — APPROVED** [review]
rest LGTM

**Pull request merged**